pkg/granted/credentialsprocess.go

```diff
@@ -0,0 +1,103 @@
+package granted
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/aws/smithy-go"
+	"github.com/common-fate/granted/pkg/cfaws"
+	"github.com/fatih/color"
+	"github.com/urfave/cli/v2"
+)
+
+/*
+* AWS Creds consumed by credential_process must adhere to this schema
+* https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-sourcing-external.html
+ */
+type AWSCredsStdOut struct {
+	Version         int    `json:"Version"`
+	AccessKeyID     string `json:"AccessKeyId"`
+	SecretAccessKey string `json:"SecretAccessKey"`
+	SessionToken    string `json:"SessionToken"`
+	Expiration      string `json:"Expiration"`
+}
+
+var CredentialsProcess = cli.Command{
+	Name:   "credentialsprocess",
+	Usage:  "",
+	Hidden: true,
+	Flags:  []cli.Flag{&cli.StringFlag{Name: "profile", Required: true}, &cli.StringFlag{Name: "url"}},
+	Action: func(c *cli.Context) error {
+
+		url := c.String("url")
+		profileName := c.String("profile")
+
+		if url == "" {
+			url = "http://localhost:3000/"
+		}
+		// In future we will support manual storing of url in config...
+		// conf, err := config.Load()
+		// if err != nil {
+		// 	log.Fatal("Failed to load Config for GrantedApprovalsUrl")
+		// }
+		// url = conf.GrantedApprovalsUrl
+		if url == "" {
+			log.Fatal("It looks like you haven't setup your GrantedApprovalsUrl\nTo do so please run: " + c.App.Name + " setup")
+		}
+
+		profiles, err := cfaws.LoadProfiles()
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		profile, err := profiles.LoadInitialisedProfile(c.Context, profileName)
+		if err != nil {
+			_, ok := err.(*smithy.OperationError)
+			if ok {
+				// if it failed to load initialised try load it from the config
+				pr, err := profiles.Profile(profileName)
+				if err != nil {
+					log.Fatal(err)
+				}
+				log.Fatal(getGrantedApprovalsUrl(url, pr))
```

Comment:
Does the output get printed if we return an error instead of calling `log.Fatal`? I would prefer to remove all calls to `log.Fatal`, as it prints an unnecessary timestamp before the message.

```suggestion
				return errors.New(getGrantedApprovalsUrl(url, pr))
```

If it doesn't work, can we replace this with:
```go
fmt.Println(getGrantedApprovalsUrl(url, pr))
os.Exit(1)
```

Reply:
Note that this comment applies to all use of `log.Fatal` here as it's being used in a few spots